app/src/main/java/fr/free/nrw/commons/contributions/ContributionsFragment.java

```diff
@@ -494,7 +493,6 @@ private void checkPermissionsAndShowNearbyCardView() {
                 && !store.getBoolean("doNotAskForLocationPermission", false)
                 && (((MainActivity) getActivity()).activeFragment == ActiveFragment.CONTRIBUTIONS)) {
             nearbyNotificationCardView.permissionType = NearbyNotificationCardView.PermissionType.ENABLE_LOCATION_PERMISSION;
-            showNearbyCardPermissionRationale();
```

Comment:
This is why the dialog is not being shown, I'll revert this.

Reply:
I could confirm the dialog pops up after you've reverted this line 👍🏼